app/components/map-form/inset-map.js

```diff
@@ -21,4 +21,19 @@ class InsetMap extends Component {
       duration: 0,
     });
   }
+
+  @observes('boundsPolygon')
+  resize() {
+    const map = this.get('mapInstance');
+    // just incase this gets triggered unexpectedly before the map has been loaded,
+    // only attempt to re-render if map exists
+    if (map) {
+      this.resizeMap(map);
+    }
+  }
```

Comment:
Observers just make more sense here, it's easier to understand what's going on. 

Sometimes, however, these properties can be updated "too fast", leading to performance hits. It's a judgement call at that point. 

We already talked about all the alternatives. Here's how the ember-mapbox-gl bindings lib handles this: https://github.com/kturney/ember-mapbox-gl/blob/master/addon/components/mapbox-gl-call.js#L22

^ using didReceiveAttrs, they more or less add logical checks for seeing _what_ was updated, etc. We're not doing anything that generic here so this seems less confusing.

I want to write a blog post about observers sometime because it's really unclear as to when they're supposed to be used. There's even a _linting rule_ against using them, but they are a critical, public part of the framework.